README.md

```diff
@@ -1,2 +1,29 @@
 # DnDeep-backend
 The backend part of DnDeep
+
+to create a Character send a POST request to http://localhost:3000/characters/ (if on local host) with a body similar to what is seen bellow:
+{
+	 "level": "10",
+    "abilityScores": [
+        "20",
+        "20",
+        "20",
+        "20",
+        "20",
+        "20"
+    ],
+    "_id": "5e42c05e0b7a2c3cc094bc90",
```

Comment:
_id is not needed, when POSTing a new character, it will be generated by Mongo/Mongoose ;)

Reply:
true, this went quite fast